commit 5f146e57fbeae9fa34ea88c0eac13bfa5eb799cb (refs/changes/99/4499/1)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2018-07-09T19:30:09-04:00 (1 year, 3 months ago)
    
    joyent/node-sshpk#51 package.json repository does not point to Joyent

diff --git a/package.json b/package.json
index af11e3b..e085195 100644
--- a/package.json
+++ b/package.json
@@ -8,7 +8,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/arekinath/node-sshpk.git"
+    "url": "git+https://github.com/joyent/node-sshpk.git"
   },
   "author": "Joyent, Inc",
   "contributors": [
